redis.conf

```diff
@@ -1449,15 +1450,16 @@ auto-aof-rewrite-min-size 64mb
 # will be found.
 aof-load-truncated yes
 
-# When rewriting the AOF file, Redis is able to use an RDB preamble in the
-# AOF file for faster rewrites and recoveries. When this option is turned
-# on the rewritten AOF file is composed of two different stanzas:
-#
-#   [RDB file][AOF tail]
-#
-# When loading, Redis recognizes that the AOF file starts with the "REDIS"
-# string and loads the prefixed RDB file, then continues loading the AOF
-# tail.
+# The BASE AOF encode type.
+# 
+# When rewriting the AOF file, Redis will generate a new INCR file and a 
+# BASE file. INCR file is always AOF (RESP) encoded , but BASE file can be 
+# in AOF format and RDB format. When this option is turned on, the BASE file
+# use RDB format (the file name uses .rdb as the suffix), otherwise use AOF 
+# format (the file name uses .aof as the suffix).
+# 
+# When loading, Redis recognizes that the BASE AOF file starts with the "REDIS"
+# string and loads the BASE RDB file, then continues loading the INCR AOF file.
```

Comment:
There was too much detail here IMHO.
users and admins don't need to understand what RESP is (that's for client library developers). and don't care about the file format header bytes.
```suggestion
# The base AOF rewrite encodeing type.
# 
# When rewriting the AOF file, Redis will generate a new incremental file and a 
# new base file. The base file can be in AOF (commands) format or RDB
# (serialized) format. When this option is turned on, the base file uses RDB
# format, otherwise it uses AOF format.

```

Reply:
Fix